source/docs/magento2/page-builder.md

```diff
@@ -1,23 +1,18 @@
 ---
 id: magento2-page-builder
 title: Page Builder
+description: Adobe Commerce and Magento 2.4.3+ allow merchants to author pages using Page Builder. Front-Commerce supports Page Builder managed content out-of-the-box. In this guide, you will learn how to use this feature and extend it.
 ---
 
-Adobe Commerce and Magento 2.4.3+ allow merchants to author pages using [Page Builder](https://magento.com/products/magento-commerce/page-builder). Front-Commerce supports Page Builder managed content out-of-the-box.
-
-In this section, you will learn how to use this feature and extend it. Create new _content types_ and refine UI components so that merchants can create the rich shopping experiences that was designed for _their_ customers.
-
 <blockquote class="feature--new">
 _Since version 2.11.0 (early preview)_
-<br>
-**Early preview:** we're currently looking for feedbacks about this feature. The core API is stable and ready for use in your storefronts. We're improving default content types support. Please contact us if you're using this feature.
-<br>
-**Work In Progress:** this documentation page will get more details in the next few days
 </blockquote>
 
+Common tasks as a developer would consist in creating new _content types_ and refining UI components so that merchants can create the rich shopping experiences that was designed for _their_ customers.
```

Comment:
```suggestion
Common tasks as a developer would consist in creating new _content types_ and refining UI components so that merchants can create the rich shopping experiences that were designed for _their_ customers.
```